x-pack/plugins/beats_management/public/lib/adapters/beats/adapter_types.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CMBeat } from '../../../../common/domain_types';
+
+export interface CMBeatsAdapter {
```

Comment:
I have defined `CMPopulatedBeat` on my Beats List branch; I assume we will add methods for getting that data here as well?